packages/anoncreds/src/services/registry/AnonCredsRegistryService.ts

```diff
@@ -0,0 +1,28 @@
+import type { AnonCredsRegistry } from '.'
+import type { AgentContext } from '@aries-framework/core'
+
+import { injectable } from '@aries-framework/core'
+
+import { AnonCredsModuleConfig } from '../../AnonCredsModuleConfig'
+import { AnonCredsError } from '../../error'
+
+/**
+ * @internal
+ * The AnonCreds registry service manages multiple {@link AnonCredsRegistry} instances
+ * and returns the correct registry based on a given identifier
+ */
+@injectable()
+export class AnonCredsRegistryService {
+  public async getRegistryForIdentifier(agentContext: AgentContext, identifier: string): Promise<AnonCredsRegistry> {
+    const registries = agentContext.dependencyManager.resolve(AnonCredsModuleConfig).registries
+
+    // TODO: should we check if multiple are registered?
```

Comment:
Ah yeah that's a good question. I am okay with picking the first one, but we have to mark it explicitly in the code documentation for the public function which uses this `getRegistryForIdentifier` under-the-hood.